From: Michael Froman <mfroman@mozilla.com>
Date: Wed, 16 Jul 2025 21:08:00 +0300
Subject: Bug 1974813 - turn off rust in libwebrtc since we don't use any
 libwebrtc rust code. r=ng?

Note: no moz.build changes are caused by this change.

Differential Revision: https://phabricator.services.mozilla.com/D257316
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/14ee98c4832a55bf32116c925540ab0f53e46cd3
---
 .gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gn b/.gn
index 4c9b95c31b..80ff8790a4 100644
--- a/.gn
+++ b/.gn
@@ -91,7 +91,7 @@ default_args = {
   use_fuzztest_wrapper = false
 
   # Enable Rust in WebRTC
-  enable_rust = true
+  enable_rust = false # Mozilla
   enable_rust_cxx = true
   enable_chromium_prelude = true
 }
